src/test/resources/feature/SignInScenarios.feature

```diff
@@ -0,0 +1,32 @@
+@Anatolie
+Feature: Sign in and Forgot password
+
+  Scenario: Login validation with correct data
+    Given User is on "Authentication" page
```

Comment:
If you have some initial steps that repeats all over the feature file (ex User is on "Authentication" page) you can move it to Background.